src/utils/helper/validationChecker.ts

```diff
@@ -0,0 +1,27 @@
+export const isAlphabetic = (value: string): boolean => {
+    const regex = /^[a-zA-Z]+$/;
+    return regex.test(value);
+  };
+  
+  export const isAlphaNumeric = (value: string): boolean => {
+    const regex = /^[a-zA-Z0-9]+$/;
+    return regex.test(value);
+  };
+  
+  export const isAlphaNumericWithSpecial = (value: string): boolean => {
+    const regex = /^[a-zA-Z0-9!@#$%^&*()_+{}\[\]:;<>,.?~\\-]+$/;
+    return regex.test(value);
+  };
+  
+  export const isValidEmail = (email: string): boolean => {
+    const regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    return regex.test(email);
+  };
+  
+  export const isStrongPassword = (password: string): boolean => {
+    console.log(password, "===<>ssdadasas");
+    
+    const regex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+    return regex.test(password);
+  };
+  
```

Comment:
pls add eof here

Reply:
What is eof stand? I didn't get this changes.

Reply:
you need to add an end of line as an enter here so last line don't show minus sign